modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/AbstractDeploymentTestCase.java

```diff
@@ -1338,6 +1359,25 @@ protected void executeApplicationFlow(String flowName, String correlationId) thr
     });
   }
 
+  protected String executeApplicationFlowAndReturnMessage(String flowName) throws Exception {
```

Comment:
can we reuse code from executeApplicationFlow. I think there is duplicate code here.